src/htmx.js

```diff
@@ -1319,7 +1361,8 @@ var htmx = htmx || (function () {
                 defaultSwapStyle:'innerHTML',
                 defaultSwapDelay:0,
                 defaultSettleDelay:100,
-                includeIndicatorStyles:true
+                includeIndicatorStyles:true,
+                defaultEncoding: "xml"
```

Comment:
I think the default should be `x-www-form-urlencoded`, not XML (it isn't XML format)

Reply:
yes, that makes sense.  Still do you really want people to type all that out.  how about:
defaultEncoding = "url";
the only other option being:
defaultEncoding = "json";
NOTE:
The code only looks for "json" anything else is automatically considered:  x-www-form-urlencoded

Reply:
Per your okay, we will go with "url".